style.css

```diff
@@ -192,19 +206,29 @@ header {
   font-family: 'Inter', sans-serif;
 }
 
+.cards {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  gap: 10px;
+}
+
 .card1 {
   display: flex;
   flex-direction: column;
   background-image: url(./img/project-img-resized.png);
-  background-size: cover;
+  background-size: contain;
   background-repeat: no-repeat;
-  margin-left: 1.5rem;
-  margin-top: 24px;
-  margin-right: 1.5rem;
   color: white;
+  height: 435px;
+  width: 369px;
+  align-items: center;
+  justify-content: center;
+  margin-bottom: 20px;
 }
```

Comment:
- [ ] Your regular project cards are great. But the 'See Project' button should only appear when the user hovers over the card, making the title, description, technologies list and dark overlay of the image disappear, and the 'See Project' button should appear at the bottom of the image like the Figma template. Kindly implement this feature to match the Figma template.
 - Your design:
 - 
<img width="1200" alt="image" src="https://user-images.githubusercontent.com/61919617/214893299-98a5eecf-29ea-43bd-98f0-d6c056fbc2a2.png">

 - Figma template:
 - 
<img width="953" alt="image" src="https://user-images.githubusercontent.com/61919617/214893434-31981bff-29f7-443e-b01d-c2198cda6011.png">